### ISSUE TITLE
test_pool_restart fails on Debian unstable

### ISSUE THREAD

**Issue by bdrung** [mher/flower#442]
The test_pool_restart test fails on Debian unstable:
```
$ python2.7 setup.py test
======================================================================
ERROR: test_pool_restart (tests.api.test_control.WorkerControlTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/tornado/testing.py", line 120, in __call__
    result = self.orig_method(*args, **kwargs)
  File "/build/flower-XP5LM3/flower-0.8.2+dfsg/tests/api/test_control.py", line 37, in test_pool_restart
    celery.control.broadcast.assert_called_once()
  File "/usr/lib/python2.7/dist-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once

----------------------------------------------------------------------
```
Installed Python packages:
python 2.7.9-1
python-all 2.7.9-1
python-amqp 1.4.6-3
python-anyjson 0.3.3-1
python-babel 1.3+dfsg.1-5
python-babel-localedata 1.3+dfsg.1-5
python-billiard 3.3.0.20-1
python-celery 3.1.18-1
python-celery-common 3.1.18-1
python-chardet 2.3.0-1
python-colorama 0.3.3-1
python-concurrent.futures 2.2.0-1
python-dateutil 2.2-2
python-distlib 0.2.1-1
python-docutils 0.12+dfsg-1
python-funcsigs 0.4-1
python-html5lib 0.999-3
python-jinja2 2.8-1
python-kombu 3.0.26-1
python-mailer 0.7-1
python-markupsafe 0.23-2
python-memcache 1.56-1
python-minimal 2.7.9-1
python-mock 1.3.0-2.1
python-pbr 0.11.0-1
python-pip 1.5.6-6
python-pkg-resources 18.0.1-2
python-pycurl 7.19.5.1-1
python-pygments 2.0.1+dfsg-1.1
python-pyparsing 2.0.3+dfsg1-1
python-requests 2.7.0-3
python-roman 2.0.0-1
python-setuptools 18.0.1-2
python-six 1.9.0-3
python-sphinx 1.2.3+dfsg-1
python-support 1.0.15
python-tornado 4.2.0-1
python-tz 2012c+dfsg-0.1
python-urllib3 1.10.4-1

**Comment by mher** [comment]
Is this sill an issue?

**Comment by bdrung** [comment]
My bad that I didn't specified the affected version. I guess it is fixed since I packaged version 0.8.3 without that test failure.
